docs/LogGuide.md

```diff
@@ -2473,6 +2473,9 @@ These lines give more granular information when combatants change their status.
 Please note that this is still polling memory (so timing may be racy)
 and there are some heuristics to not emit too many lines (so data may be imprecise).
 
+For more information,
+see the [class definition](https://github.com/OverlayPlugin/OverlayPlugin/blob/main/OverlayPlugin.Core/MemoryProcessors/Combatant/LineCombatant.cs#L27) in OverlayPlugin.
+
 There are three types of this line:
 
 - Add: emits all initial fields for this combatant
```

Comment:
Just thought of this today, but it may also make sense to note that `Add` lines exclude "default" values, e.g. if it's an `int` value type and the value is `0` on add, it won't write that field?

Reply:
Oh yeah, that's really important to say too, thank you!